html/kalendarium.html

```diff
@@ -36,10 +36,29 @@
     <main>
 
         <ul class="breadcrumbs">
-            <li>Startsida / </li>
-            <li></li>
+            <li><a href="/index.html">Startsida</a> / </li>
+            <li>Kalendarium</li>
         </ul>
 
+        <h1>Aktuellt i kalendariet</h1>
+
+        <p>Här kan du se alla aktuella aktiviteter inom myndigheten.</p>
+
```

Comment:
Wrap these in `<section>`.

Reply:
correct, it is what will be on the "hero"